DenzelleJinx/ECSE-428-Fall-2024
# View My Listing as a landlord

The purpose of this is landlord are now able to see their own listings in a page called My Listings
To test it: 
create multiple listings from different landlords
Log in as one landlord (with at least one listing to his/her/their name) and navigate to the page My Listings
Verify if you can view Listings for specific landlord

Repeat the same step with a landlord without listing, you should still see the page My Listing with the search bar and the page mentions that you currently have no listing
